_posts/2022-11-02-relicensing.md

```diff
@@ -3,31 +3,63 @@ layout: post
 title: Relicensing MDAnalysis
 ---
 
+<a href="https://www.gnu.org/licenses/lgpl-3.0.en.html">
 <img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
 title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
+</a>
+
+This blog post outlines MDAnalysis' proposal to change its license
+to the [Lesser GNU Public License][LGPL] (LGPL v3+).
+
+A summary of our [reasons for proposing this license
+change](#Rationale-for-changing-licenses), alongside upcoming actions
+for [community members and library
+contributors](#How-will-the-relicensing-process-work?) are provided.
+
+
+> ⚠️ **Disclaimer**
+> The MDAnalysis core team members are not
+> lawyers, as such the information provided here __does not, and is not
+> intended to, constitute legal advice__. This blog post also does not
+> represent MDAnalysis' full legal position on software licensing, it
+> purely aims to inform MDAnalysis developers and users on why
+> we believe the library should be relicensed.
+>
+> Further information on open-source software licensing can be found
+> from sources such as the [Open Source Initiative][OSI],
+> [tl;drLegal][tldr legal] and the [Software Sustainability Insitute][SSI licensing].
+>
+> Should you have any concerns about licensing, we always strongly
+> recommend getting legal advice before making any decisions on how
+> licensing changes may affect you.
+
+
+## Overview
 
 We want to **change the license of MDAnalysis** from the [GNU General
-Public License][GPLv2] (GPL v2+) to the more permissive [Lesser GNU
-Lesser Public License][LGPL] (LGPL v3+) license. Both are *open source
-licenses* but the [LGPL][] gives developers more freedom in how they
-license any of their own code that makes use of MDAnalysis.
+Public License v2 (or any later versions)][GPLv2] (GPL v2+) to the less
+restrictive [Lesser GNU Lesser Public License v3 (or any later versions)][LGPL]
+(LGPL v3+) license. Both are [_open source licenses_][OSI] but it is our view
+that the [LGPL][] will give developers more freedom in how they license any of
+their own codes that makes use of MDAnalysis.
 
-The license is the legal basis that determines how code can be copied,
-changed, and incorporated into other code. The license is the only
-reason that one can use someone else's code --- it does not matter if
-the code is publicly available somewhere. If it does not have a
-license you cannot use it legally. Therefore, getting the license
-right is very important.
+As detailed by the [Open Source Definition][OSD], licenses are core to the
```

Comment:
Took some time, but settled on the OSD as the authoritative "this is why licenses are important" viewpoint.